src/fontra/views/editor/panel-transformation.js

```diff
@@ -450,6 +579,452 @@ export default class TransformationPanel extends Panel {
     this.update();
   }
 
+  _getObjectBounds(layerGlyphController, selection) {
+    if (selection.startsWith("contour")) {
+      const contour = selection.split("/")[1];
+      const points = contour.split(",").map((pointIndex) => `point/${pointIndex}`);
+      return layerGlyphController.getSelectionBounds(points);
+    }
+
+    return layerGlyphController.getSelectionBounds([selection]);
+  }
+
+  _getSortedObjects(
+    layerGlyphController,
+    points,
+    contours,
+    components,
+    distributeDirection
+  ) {
+    let objects = [];
+    for (const pointIndex of points) {
+      objects.push(`point/${pointIndex}`);
```

Comment:
I build strings, because I need them as 'individual selections' to pass to EditBehaviorFactory. Either I build them here or later. But I have to do it, at some point.

Reply:
But you also introduce `comtour/<index>`, which doesn't exist as a selection item.

Then there's this:
```
          if (object.startsWith("contour")) {
            const pointIndices = object
              .split("/")[1]
              .split(",")
              .map((pointIndex) => parseInt(pointIndex));
```
There is literally no reason to represent these things as strings. Sure, at the _end_ when you really need a Fontra selection set, then, but not earlier.

Reply:
OK, let me change it.

Reply:
I changed it. The 'string selection' happens later + in a separate method. I won't mark this feedback as resolved, because I would like to get your feedback first.